rest/tests/test_rest.py

```diff
@@ -0,0 +1,742 @@
+"""
+Unit tests for REST api
+"""
+
+from __future__ import unicode_literals
+
+import json
+from django.test import Client
+from rest_framework.reverse import reverse
+
+from learningresources.tests.base import LoreTestCase
+from learningresources.models import Repository
+from taxonomy.api import create_vocabulary
+from taxonomy.models import Vocabulary, Term
+from rest.serializers import VocabularySerializer
+
+OK = 200
+CREATED = 201
+NO_CONTENT = 204
+BAD_REQUEST = 400
+FORBIDDEN = 403
+NOT_FOUND = 404
+METHOD_NOT_ALLOWED = 405
+
+
+def as_json(resp):
+    """Get JSON from response"""
+    return json.loads(resp.content.decode('utf-8'))
+
+
+class TestRest(LoreTestCase):
+    """
+    REST test
+    """
+    # pylint: disable=too-many-public-methods, too-many-arguments
+    def get_repositories(self, expected_status=OK):
+        """Get list of repositories"""
+        resp = self.client.get('/api/v1/repositories/')
```

Comment:
since you use this base string a lot of times, probably it's better to declare it in the setUp of this test class and reuse it